client/elements/addons/text-carousel.html

```diff
@@ -89,14 +90,26 @@
 
     </style>
 
+    <iron-ajax
+        auto
```

Comment:
is `auto` the best here?

Reply:
Good point. I'll check tomorrow if it hinders performance.